src/components/CaptureViewer/index.tsx

```diff
@@ -1,19 +1,34 @@
-import { h } from 'preact'
+import { h, FunctionComponent } from 'preact'
 import { isOfMimeType } from '~utils/blob'
 import PdfViewer from './PdfViewer'
 import CaptureImageViewer from './CaptureImageViewer'
 import CaptureVideoViewer from './CaptureVideoViewer'
 
-const CaptureViewer = ({
+import type { CaptureMethods } from '~types/commons'
+import type { CapturePayload } from '~types/redux'
+
+type Props = {
+  capture: CapturePayload
+  method: CaptureMethods
+  isFullScreen?: boolean
+  imageAltTag?: string
+  videoAriaLabel?: string
+}
+
+const CaptureViewer: FunctionComponent<Props> = ({
   capture: { blob, id, variant, isPreviewCropped },
   method,
   isFullScreen,
   imageAltTag,
   videoAriaLabel,
 }) => {
-  if (isOfMimeType(['pdf'], blob)) return <PdfViewer blob={blob} />
-  else if (variant === 'video')
-    return <CaptureVideoViewer ariaLabel={videoAriaLabel} blob={blob} />
+  if (isOfMimeType(['pdf'], blob)) {
+    return <PdfViewer blob={blob} />
+  }
+
+  if (variant === 'video') {
+    return <CaptureVideoViewer ariaLabel={videoAriaLabel || ''} blob={blob} />
```

Comment:
I think for accessibility there should always be a `videoAriaLabel` string even if just a default/generic one if it isn't defined. Something like "Video preview" in this case.

Reply:
Added `Video preview` as default fallback for `videoAriaLabel`